calling/src/commonMain/kotlin/com/wire/kalium/calling/CallType.kt

```diff
@@ -0,0 +1,10 @@
+package com.wire.kalium.calling
+
+/**
+ * AUDIO for audio call
+ * VIDEO for video cal
```

Comment:
We can use Kdoc features here to add links to the definitions.

```suggestion
 * [AUDIO] for audio call
 * [VIDEO] for video cal
```

In the IDE (pressing F1 on Mac, or Ctrl+Q on Windows/Linux) or in Dokka, they are supported.